docs/docs/start-here.md

```diff
@@ -0,0 +1,106 @@
+---
+sidebar_position: 1
+---
+
+# Start Here
+
+## What is this?
+
+[A group of organizations](about) are working together to define a common set of
+annotation types for use in Java static analysis, beginning with nullness
+analysis.
+
+This includes providing both a JAR file your code can depend on, and *precise*
+specifications of these annotations' semantics.
+
+* **Why standardize the annotations?** This tragic [stackoverflow answer] says
+  it all.
+* **Why standardize the semantics?** Because a library owner, or a team that
+  uses multiple tools, deserves a single source of truth for how to
+  annotate their code, instead of having to decide which tool to make happy.
+* **Why do both together?** Because users deserve annotation types with
+  Javadoc *right there* that answers their questions authoritatively.
+
+Our release will be the first tool-neutral, library-neutral artifact for these
+annotations; it is being developed by consensus of major stakeholders in Java
+static analysis.  (Note: `javax.annotation` was an aborted attempt at this that
+was never actually released, whose draft work was jammed into Maven anyway.)
+
+Learn more about the JSpecify group and its goals in the [JSpecify FAQ].
+
+## How do I learn about your nullness support?
+
+As you read the following, when you think "that's weird, why did they do
+that?", have a look at the [Nullness Design FAQ] to see if your question has
+been answered.
+
+### Begin with one of
+
+* The [Javadoc], which is reasonably thorough and specific, though not an
+  organized walkthrough
+* The current [User Guide] draft, which is a better choice for reading
+  top-to-bottom, but is currently a bit outdated and in need of improvement.
+  It will change quite a bit before 1.0.
+
+### Then if you're *really* interested
+
+* The [specification], written to be understood by owners of compilers and
+  static analysis tools, and currently a bit outdated
```

Comment:
Probably better to mention that it's outdated only on the spec itself and just point to it here. If we say that it's outdated here we'll need to remember to go back and also update this doc when it's no longer outdated.

Reply:
Fair point, but... I do feel better warning about this right here. So I filed #344 with a reminder about this.